Penteract/Assets/Scripts/HUDController.cpp

```diff
@@ -482,32 +489,32 @@ void HUDController::AbilityCoolDownEffectCheck(Cooldowns cooldown, GameObject* c
 
 				if (cooldown < Cooldowns::ONIMARU_SKILL_1) {
 					if (canvas->GetChildren().size() > 0) {
-						if (canvas->GetChildren()[static_cast<int>(cooldown)]->GetChildren().size() > 5) {
-							ef = GET_SCRIPT(canvas->GetChildren()[static_cast<int>(cooldown)]->GetChildren()[0], AbilityRefreshEffect);
+						if (canvas->GetChildren()[static_cast<int>(cooldown)]->GetChildren().size() > HIERARCHY_INDEX_MAIN_BUTTON_DOWN) {
```

Comment:
Main button down is 6 and the old value is 5. Is this correct?

Reply:
Nicely spotted